server/grpc/server.go

```diff
@@ -0,0 +1,50 @@
+package grpc
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+
+	"github.com/cosmos/cosmos-sdk/server/types"
+)
+
+const (
+	// GRPCBlockHeightHeader is the gRPC header for block height.
+	GRPCBlockHeightHeader = "sdk-block-height"
+	// GRPCWithProofHeader is a gRPC header which is true when the request
+	// requires a proof.
+	GRPCWithProofHeader = "sdk-require-proof"
```

Comment:
I'm not sure we should really have this because we can't actually support proofs with gRPC yet and I don't want to set false expectations. It would be nice to support it in the future, but it would need some serious architecture work.